src/tests_cfg/cake_filling_price.rs

```diff
@@ -22,13 +22,17 @@ pub struct Model {
     pub price: Decimal,
     #[sea_orm(ignore)]
     pub ignored_attr: i32,
+    pub indexed_attr: i32,
```

Comment:
I would very much appreciate that we do not add columns to this Entity.
May be we can test it by having multiple test cases, with each test a different behaviour

Reply:
In the update, I have created a new standalone Entity for the purpose of testing index creation